docs/webpack.md

```diff
@@ -78,6 +78,31 @@ babelLoader.options.cacheDirectory = false
 module.exports = environment
 ```
 
+### Overriding Loader Options in Webpack 3+ (for CSS Modules etc.)
+
+In Webpack 3+, if you'd like to specify additional or different options for a loader, editing the actual `config/webpack/{loader}.js` file is *not* the way to go.
```

Comment:
Lets remove reference to this `config/webpack/{loader}.js`, since it no longer exists.  All base loaders are inside npm module now.